qutip/__init__.py

```diff
@@ -60,6 +59,13 @@
 from .about import *
 from .cite import *
 
+
+def orbital(*_, **__):
```

Comment:
I'm happy for us to just drop `orbital`. We can add this note to the release notes instead.